webapp/channels/src/components/center_message_lock/index.tsx

```diff
@@ -111,28 +111,40 @@ export default function CenterMessageLock(props: Props) {
             defaultMessage: 'Unlock messages prior to {date} in {team}',
         }, titleValues);
 
-        description = intl.formatMessage(
-            {
-                id: 'workspace_limits.message_history.locked.description.admin',
-                defaultMessage: 'To view and search all of the messages in your workspace’s history, rather than just the most recent {limit} messages, upgrade to one of our paid plans. <a>Review our plan options and pricing.</a>',
-            },
-            {
-                limit,
-                a: (chunks: React.ReactNode | React.ReactNodeArray) => (
-                    <a
-                        href='#'
-                        onClick={(e: React.MouseEvent) => {
-                            e.preventDefault();
-                            openPricingModal({trackingLocation: 'center_channel_posts_over_limit_banner'});
-                        }}
-                    >
-                        {chunks}
-                    </a>
-                ),
-            },
-        );
-
-        cta = (
+        if (isAirGapped) {
+            description = intl.formatMessage(
+                {
+                    id: 'workspace_limits.message_history.locked.description.admin.airgapped',
+                    defaultMessage: 'To view and search all of the messages in your workspace\'s history, rather than just the most recent {limit} messages, upgrade to one of our paid plans.',
```

Comment:
This component and logic of message limits can probably be deprecated as well but probably out of scope for this PR